requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Runtime requirements for the python 3 version of cirq.
 
-dataclasses; python_version < '3.8'
+dataclasses; python_version < '3.9'
```

Comment:
you know...actually this is silly. I modified it for 3.7 as well. But actually it is a version modifier for the dataclasses package, which means "only install this if python version is less than 3.9" - which doesn't make sense as dataclasses are included since 3.7 in python. So this is backport thing for python 3.6. I'll fix it. Great catch :D